WcaOnRails/config/locales/de.yml

```diff
@@ -2844,6 +2854,16 @@ de:
       orga_closed_reg_success: Anmeldung erfolgreich geschlossen.
       #original_hash: 7f197a0
       orga_closed_reg_failure: Anmeldung kann nicht geschlossen werden.
+      #original_hash: b16bd13
+      cannot_activate_v2: >-
+        Du kannst das neue Anmeldesystem nicht aktivieren, wenn die Anmeldung
+        geöffnet ist.
+      #original_hash: 74caf1a
+      no_waiting_list_specified: >-
+        Dieser Wettbewerb hat keine Deadline zum Akzeptieren von Teilnehmern auf
```

Comment:
Frist, s.o.

Reply:
Außerdem: Teilnehmenden. Gendert die restliche Übersetzungsdatei?

Reply:
Tatsächlich haben wir wohl bis dato nichts gegendert, aber das werden wir, nach Rücksprache mit den anderen, wohl jetzt einführen. Vermutlich in einem mit den Regs / Guidelines. Das würde ich dann allerdings trennen und einen zweiten PR für alle Vorkommen hier machen, wenn das OK ist?